backend/src/seeds/factory/creation.ts

```diff
@@ -17,19 +17,19 @@ export const nMonthsBefore = (date: Date, months = 1): string => {
 export const creationFactory = async (
   client: ApolloServerTestClient,
   creation: CreationInterface,
```

Comment:
rename the CreationInterface to ContributionInterface

Reply:
You are right. Please let us keep this for later ...